### PR TITLE
Add revenue breakdowns

### PR DESCRIPTION
**Pull request by nikhilwoodruff**
Screenshots:
## Non-expanded by default
image
## Expanding (with a time estimate)
image
## Expanded
image

### CODE REVIEW

File: policyengine/countries/country.py [hunk 1]
```diff
@@ -183,3 +187,12 @@ def variables(self, params=None):
     @exclude_from_cache
     def default_household(self, params=None):
         return self.default_household_data
+
+    def population_breakdown(self, params=None):
+        reform, provisions = create_reform(
+            params, self.policyengine_parameters, return_descriptions=True
+        )
+        breakdown = get_breakdown_per_provision(
```
Comment (MaxGhenis) [inline policyengine/countries/country.py:195]:
```suggestion
        return get_breakdown_per_provision(
```

File: policyengine-client/src/common/advanced.jsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    for(let i = 0; i < props.results.spending.length; i++) {
+        data.push({
+            key: props.results.provisions[i],
+            provision: props.results.provisions[i],
+            additional_spending: props.results.spending[i],
+            cumulative_spending: props.results.cumulative_spending[i],
+        });
+    }
+    return <Table columns={columns} dataSource={data} />;
+}
```
Comment (MaxGhenis) [inline policyengine-client/src/common/advanced.jsx:80]: newline

**Review by MaxGhenis — COMMENTED** [review]
could you drop a screenshot in?

File: policyengine/countries/country.py (hunk 1 above)
Comment (nikhilwoodruff) [reply policyengine/countries/country.py]: Done

File: policyengine-client/src/common/advanced.jsx (hunk 2 above)
Comment (nikhilwoodruff) [reply policyengine-client/src/common/advanced.jsx]: Added

**Pull request merged**